mantis-common/src/main/java/io/mantisrx/runtime/descriptor/StageSchedulingInfo.java

```diff
@@ -38,21 +41,29 @@ public class StageSchedulingInfo implements Serializable {
     private final StageScalingPolicy scalingPolicy;
     private final boolean scalable;
 
+    /**
+     * Nullable field to store container attributes like sku ID assigned to this stage.
+     */
+    @JsonInclude(Include.NON_NULL)
+    private final Map<String, String> containerAttributes;
+
     @JsonCreator
     @JsonIgnoreProperties(ignoreUnknown = true)
     public StageSchedulingInfo(@JsonProperty("numberOfInstances") int numberOfInstances,
                                @JsonProperty("machineDefinition") MachineDefinition machineDefinition,
                                @JsonProperty("hardConstraints") List<JobConstraints> hardConstraints,
                                @JsonProperty("softConstraints") List<JobConstraints> softConstraints,
                                @JsonProperty("scalingPolicy") StageScalingPolicy scalingPolicy,
-                               @JsonProperty("scalable") boolean scalable) {
+                               @JsonProperty("scalable") boolean scalable,
+                               @JsonProperty("containerAttributes") Map<String, String> containerAttributes) {
```

Comment:
did you verify if this can deserialize old jsons?

Reply:
There is test covering the old payload.